### DISCUSSION TITLE
Best way to share OpenHAB MQTT configuration for Shelly2.5 running tasmota?

### DISCUSSION THREAD

**Discussion by Entepotenz** [arendst/Tasmota#14749] [Q&A] [answered]
Hi everyone,

I flashed a **Shelly2.5** with Tasmota and invested some time in creating the **MQTT** bindings, items, things..... for **OpenHAB**.
I would like to share the result with the community so everyone can just copy&paste and do not rewrite it again.

I guess some of those values are Shelly2.5 specific which is why I think the https://tasmota.github.io/docs/openHAB/#mqttv2-integration documentation is not the right place to put this.
**Anyone with an idea where / how to add my configuration templates for Shelly 2.5?**

**Comment by barbudor** [comment, marked as answer]
IMHO the OpenHAB setion seems a good place.
May be start a kind of "Cookbook" block for various devices ?

**Reply by Entepotenz** [reply]
Ok, I will do that 👍
